src/app/public/modules/select-field/types/select-field.ts

```diff
@@ -1,6 +1,23 @@
-export interface SkySelectField {
+  /**
+   * Extends the `any` type and supports values for data objects that you can inject into
+   * the select field component to populate the picker.
+   */
+  export interface SkySelectField {
+  /**
+   * Specifies an ID for an item.
+   */
   id?: string;
+  /**
+   * Specifies an ID for an item.
+   */
   label?: string;
+  /**
+   * Specifies a category for an item.
+   */
   category?: string;
+  /**
+   * Specifies a description to display below an item's label when `selectMode`
```

Comment:
Is this description correct for this property? In the current docs at https://developer.blackbaud.com/skyux/components/select-field#skyselectfield-properties, we have a `description` property listed. The other three properties match up, so I copied the description for the `description` property in the current docs here thinking that maybe the property was just renamed at some point or that the docs were mislabeled from the get-go.

Reply:
You've uncovered a problem with the source code, unfortunately. I'll create a separate PR to address it. Thanks!

Reply:
I've fixed the issue with a previous pull request and fixed any merge conflicts in this branch. We should be good now.